src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -85,23 +85,23 @@
     @# FIXME: Convert argv from strings to the correct types.
     parser = argparse.ArgumentParser()
     @join param : sample.initCode.argDefaultParams
-      parser.add_argument('--{@param.identifier}')
+      parser.add_argument('--{@param.identifier}', default='')
```

Comment:
Update the `FIXME` above to (a) not refer to `argv` directly and (b) remind us that we will fill in the appropriate default values.

Reply:
(not sure whether you saw this; it got accidentally "resolved" by someone)

Also, I just noticed that this is tagged inconsistently. We are using for `@# TODO` for user-visible comments of things the developer needs to do, and `# FIXME` for things we ourselves need to fix in the templates. Could you s/`@# FIXME`/`# FIXME`/ above?

Reply:
Oh yeah you're right. Done.